docs/user_guide.md

```diff
@@ -1,9 +1,26 @@
 @mainpage
 
-## UFS_UTILS
+# UFS_UTILS
 
 Utilities for the NCEP models. This is part of the
 [NCEPLIBS](https://github.com/NOAA-EMC/NCEPLIBS) project.
 
-The UFS_UTILS code here: https://github.com/NOAA-EMC/UFS_UTILS.
+The UFS_UTILS code can be found here:
+https://github.com/NOAA-EMC/UFS_UTILS.
+
+## The Utilities
+
+- exemcsfc_global_sfc_prep.sh
```

Comment:
This script runs the emcsfc_ice_blend and emcsfc_snow2mdl programs.  Is it a utility in itself?

Reply:
You tell me...

Reply:
I would consider that a script and not a utility.

Reply:
I removed it from the list.